.github/workflows/scheduled-package-validation.yaml

```
@@ -0,0 +1,95 @@
+        run: pnpm install -r --no-frozen-lockfile
+        
+      - name: Compile TypeScript
+        run: pnpm run build
+        
+      - name: Run Package Validation Report
+        id: validation
+        run: |
+          node scripts/generate-package-report.js --verbose --report-only --output=validation-report.json > validation-report.txt 2>&1
+          echo "validation_exit_code=$?" >> $GITHUB_OUTPUT
+        continue-on-error: true
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)